R/dx.R

```diff
@@ -1,47 +1,66 @@
 #' @title Determine if valid dp repository
-#' @description Looks at the path, runs `dp_repository_check` and returns TRUE 
+#' @description Looks at the path, runs `dp_repository_check` and returns TRUE
 #' if all TRUE
-#' @details All diagnostic tests to check validity of dp repository are run 
-#' regardless of choice of checks. Checks determines what subset is considered 
-#' in return T/F. Regardless of the choice of checks, verbose = T
+#' @details All diagnostic tests to check validity of dp repository are run
+#' regardless of choice of checks. Checks determines what subset is considered
+#' in return T/F.
 #' @param path Path to be evaluated
 #' @param checks any combination of c("all","git","dp","renv","branch"). default
 #' is all.
 #' @param verbose If TRUE, it will print which tests passed/failed
 #' @return TRUE or FALSE
 #' @export
-is_valid_dp_repository <- function(path, 
-                                   checks = c("all","git","dp","renv","branch"),
-                                   verbose = F){
-  checks <- match.arg(arg = checks, 
-                      choices = c("all","git","dp","renv","branch"),
-                      several.ok = T)
-  
+is_valid_dp_repository <- function(path,
+  checks = c("all","git","dp","renv","branch"),
+  verbose = F){
+  checks <- match.arg(arg = checks,
+    choices = c("all","git","dp","renv","branch"),
+    several.ok = T)
+
   dx <- dp_repository_check(path = path)
-  
-  if(verbose)
-    print(data.frame(dx))
-  
 
   if(!"all" %in% checks){
     if(!"git" %in% checks)
-      dx <- dx[,setdiff(colnames(dx),"git_initialized"), drop = F]
-    
+      dx <- dx[setdiff(names(dx),"git_initialized")]
+
     if(!"dp" %in% checks)
-      dx <- dx[,setdiff(colnames(dx),"dp_initilized"), drop = F]
-    
+      dx <- dx[setdiff(names(dx),"dp_initialized")]
+
     if(!"renv" %in% checks)
-      dx <- dx[,setdiff(colnames(dx),"renv_initialized"), drop = F]
-    
+      dx <- dx[setdiff(names(dx),"renv_initialized")]
+
     if(!"branch" %in% checks)
-      dx <- dx[,setdiff(colnames(dx),"branch_name_matches"), drop = F]
+      dx <- dx[setdiff(names(dx),"branch_name_matches")]
   }
-   
+
+  if(verbose)
+    print(data.frame(dx))
+
   dp_repository <- all(sapply(dx,isTRUE))
   return(dp_repository)
-  
 }
 
+#' @title Determine if dp repository initiated
+#' @description Looks at the path, runs `dp_repository_check` and returns TRUE
+#' if any TRUE
+#' @details All diagnostic tests to check validity of dp repository are run
```

Comment:
This seems irrelevant as `is_dp_initiated` doesn't take any parameters other than `path` and `verbose`. Please correct the comment